<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Folder..</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&amp;Project Name</string>
         </property>
         <property name="buddy">
          <cstring>projectNameQLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="projectNameQLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&amp;Select a &lt;FOLDER&gt; with the following structure:

└───&lt;FOLDER&gt;
      ├───&lt;SUBFOLDER_1&gt;
      │  ├───file
      │  └───another_file
      └───&lt;SUBFOLDER_2&gt;
             ├───file
             └───another_file

</string>
         </property>
         <property name="buddy">
          <cstring>selectFolderButton</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="SelectFolderQPushButton" name="selectFolderButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select &lt;Folder&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectFolderQPushButton</class>
   <extends>QPushButton</extends>
   <header>button.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>projectNameQLineEdit</tabstop>
  <tabstop>selectFolderButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
